### ISSUE TITLE
Bank transaction id 

### ISSUE THREAD

**Issue by rezo552** [szelpe/woocommerce-barion#44]
Is it possible to retrieve the Bank approval id (Banki engedély kód?)? I need to feed it to another billing system which is expecting that => engszam varchar(10) Engedélyszám

In Woocommerce I can only retrieve the Barion generated id:

            [payment_method] => barion
            [payment_method_title] => Átutalás
            [transaction_id] => 3c1cedba3583ebxxxxxxxxxxxxx
            [date_paid] => 2021-03-12T14:21:46
            [date_paid_gmt] => 2021-03-12T13:21:46
            [date_completed] => 2021-03-22T08:52:51
            [date_completed_gmt] => 2021-03-22T07:52:51
            [cart_hash] => d46e6ee5fc2fexxxxxxxxxxxxxxxxx
            [meta_data] => Array
             

Thanks
Zoltan

**Comment by szelpe** [comment]
Hi,

You can check out the Barion docs on the available data received from Barion: https://docs.barion.com/Payment-Start-v2

**Comment by rezo552** [comment]
Yeah, I cant find it in the API description though when I use Barion as a customer it is in the confirmation email...

![image](https://user-images.githubusercontent.com/43187470/112611253-e14c4580-8e1d-11eb-8f2e-70f7c9b718c8.png)

**Comment by koikaam** [comment]
Szia Péter!
Megint én. E-mailben sem tudlak elérni és igazából sehogysem. :D Remélem minden rendben Veled! Vettünk egy bővítményt Tőled (WooCommerce Subscriptions Barion plugin) és szeretnénk, ha a frissítést nekünk is elküldenéd. Úgy láttam a WP fórumon, hogy másnak elküldted, így csak remélem, hogy nem leszünk kivételek. :) Köszönöm!
